src/Instance/InstanceDetails/InstanceAcquisition/InstanceAcquisition.js

```diff
@@ -1,66 +1,90 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { Link } from 'react-router-dom';
 
 import { useStripes } from '@folio/stripes/core';
-import {
-  Accordion,
-  MultiColumnList,
-  NoValue,
-} from '@folio/stripes/components';
+import { Accordion } from '@folio/stripes/components';
 
-import { getDateWithTime } from '../../../utils';
 import { useControlledAccordion } from '../../../common';
+import { isUserInConsortiumMode } from '../../../utils';
```

Comment:
I know you didn't put `isUserInConsortiumMode` in `utils`, but it doesn't belong there; it belongs in `stripes-core` because there is _nothing_ inventory-specific about it. Note also that it is named incorrectly; `isUserInConsortiumMode` does not inspect the user, it inspects whether an interface is available on this tenant. 

Please think about this kind of thing when you encounter code like this: is it correct to do this work here, or should this be lifted up to a higher level? This will make stripes-core more useful and reduce code duplication by centralizing common features.